src/lib/plugins/redstone.js

```diff
@@ -5,14 +5,18 @@ module.exports.server = function (serv, { version }) {
 
   const redstoneWireType = mcData.blocksByName.redstone_wire.id
   const redstoneTorchType = mcData.blocksByName.redstone_torch.id
-  const unlitRedstoneTorchType = mcData.blocksByName.unlit_redstone_torch.id
-  const poweredRepeaterType = mcData.blocksByName.powered_repeater.id
-  const unpoweredRepeaterType = mcData.blocksByName.unpowered_repeater.id
+  const repeaterType = mcData.blocksByName.repeater.id
 
+  let poweredRepeaterType, unpoweredRepeaterType, unlitRedstoneTorchType;
+  if (!serv.supportFeature('theFlattening')) {
+    unlitRedstoneTorchType = mcData.blocksByName.unlit_unlit_redstone_torch.id
+    poweredRepeaterType = mcData.blocksByName.powered_repeater.id
+    unpoweredRepeaterType = mcData.blocksByName.unpowered_repeater.id
+  }
   const powerLevel = (block, dir) => {
     if (block.type === redstoneWireType) return block.metadata
     if (block.type === redstoneTorchType) return 15
-    if (block.type === poweredRepeaterType) {
+    if (block.type === poweredRepeaterType || (block.type === repeaterType && block.metadata.powered == true)) {
```

Comment:
`block.metadata` is a number, it doesn't have a powered property

Reply:
done